client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -110,6 +111,9 @@ public static FileSystem getHdfsFs(CelebornConf conf) {
     return hdfsFs;
   }
 
+  public abstract PartitionLocation registerMapPartitionTask(
```

Comment:
Looks like there is no place using the shuffle client instance directly, why should this method be added here?

Reply:
VisableForTesting or Writer can use this directly.